tests/test_plugin_features.py

```diff
@@ -476,6 +476,34 @@ def test_pf_with_no_feature_label():
     assert label == "Plugin feature {}".format(feature_name)
 
 
+def test_pf_with_user_config_section():
+    """Use a plugin feature with the user defined config section"""
+    # setup a plugin method that uses user config section
+    def compute_with_user_section(rtdc_ds):
+        if rtdc_ds.config["user"]["channel"]:
+            area_of_region = rtdc_ds["area_um"] * \
+                             rtdc_ds.config["user"]["n_constrictions"]
+        else:
+            raise ValueError("ds.config['user']['channel'] must be True")
+        return {"area_of_region": area_of_region}
+
+    info = {"method": compute_with_user_section,
+            "feature names": ["area_of_region"]}
+    _ = PlugInFeature("area_of_region", info)
```

Comment:
General remark: If you IDE complains if you omit `_ = `, then I would say it is ok to leave it there. Personally I think it makes the code less readable.